src/components/Confirm/Confirm.js

```diff
@@ -82,39 +89,34 @@ class Confirm extends Component {
 
     actions.setCaptureMetadata({ capture, apiResponse })
 
-    const imageQualityWarning = this.getImageQualityWarningFromResponse(
-      apiResponse
-    )
+    const imageQualityWarnings = this.handleImageQualityWarning(apiResponse)
 
-    if (!imageQualityWarning) {
+    if (!imageQualityWarnings?.length) {
```

Comment:
Minor: I think we can avoid needing to use optional chaining by always returning an empty array instead of `null` at line 107.